.gitignore

```diff
@@ -107,6 +107,7 @@ Pipfile.lock
 .mypy_cache/
 
 tags
+!python_modules/dagster/dagster/_core/definitions/tags
```

Comment:
what's this?

Reply:
We gitignore all folders named `tags` right now, I want to not gitignore this module